script.py

```diff
@@ -0,0 +1,52 @@
+import requests
+import ipaddress
+import json
+
+
+def is_valid_ip(ip_str):
+    try:  
+        ip_obj = ipaddress.ip_address(ip_str) 
+        return True  
+    except ValueError:  
+        pass
+    return False
+
+with open('input.json') as file_object:
+    list_of_data = json.load(file_object)
+    for data in list_of_data:
+        URL = data['url']
+        OUTPUT_FILE = data['file']
```

Comment:
Don't use SCREAMING_SNAKE_CASE for dynamic variables, only for constants. Rename to "output_file"